docs/src/systems/boundary.md

```diff
@@ -234,3 +234,86 @@ Pages = [joinpath("schemes", "boundary", "monaghan_kajtar", "monaghan_kajtar.jl"
 - Alireza Valizadeh, Joseph J. Monaghan. "A study of solid wall models for weakly compressible SPH."
   In: Journal of Computational Physics 300 (2015), pages 5–19.
   [doi: 10.1016/J.JCP.2015.07.033](https://doi.org/10.1016/J.JCP.2015.07.033)
+
+# [Open Boundary System](@id open_boundary)
+
+TODO: Make the entire doc nicer and clearer
```

Comment:
If you want to merge it like this, please remove this TODO and add an issue instead. Or do you want to revise it in this PR?